### PR TITLE
fix(YouTube): Use correct `description`, `title` & `summary` for `URL-redirect` patch

### PR DESCRIPTION
**Pull request by johnconner122**

### CODE REVIEW

**Comment by Ushie** [comment]
Doesn't this change make the purpose of the setting more confusing?

**Comment by johnconner122** [comment]
> What's the purpose of this PR?

This patch was designed to bypass youtube.com/redirect, but changed its description to open links in browser while refactoring.

**Comment by Ushie** [comment]
Does this change help the user understand the function of this patch better?

**Comment by johnconner122** [comment]
I guess, yes.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]
Couple inconsistencies, these descriptions are used interchangeably:

- Redirect
- Redirects
- youtube.com/redirect

Additionally, the tense changes. Sometimes the descriptions are in present sometimes in future tense.

Please chose either in favour of consistency.

File: src/main/kotlin/app/revanced/patches/youtube/misc/openlinksdirectly/patch/OpenLinksDirectlyPatch.kt [hunk 1]
```diff
@@ -37,10 +37,10 @@ class OpenLinksDirectlyPatch : BytecodePatch(
         SettingsPatch.PreferenceScreen.MISC.addPreferences(
             SwitchPreference(
                 "revanced_uri_redirect",
-                StringResource("revanced_uri_redirect_title", "Open YouTube links inside app"),
+                StringResource("revanced_uri_redirect_title", "Bypass URL redirect"),
                 true,
-                StringResource("revanced_uri_redirect_summary_on", "Links opened inside YouTube ReVanced"),
-                StringResource("revanced_uri_redirect_summary_off", "Links opened in web browser")
+                StringResource("revanced_uri_redirect_summary_on", "Links will bypass youtube.com/redirect"),
+                StringResource("revanced_uri_redirect_summary_off", "Links will follow default redirect policy")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/openlinksdirectly/patch/OpenLinksDirectlyPatch.kt:43]:
```suggestion
                StringResource("revanced_uri_redirect_summary_on", "Bypassing URL redirects"),
                StringResource("revanced_uri_redirect_summary_off", "Following default redirect policy")
```

File: src/main/kotlin/app/revanced/patches/youtube/misc/openlinksdirectly/patch/OpenLinksDirectlyPatch.kt [hunk 2]
```diff
@@ -37,10 +37,10 @@ class OpenLinksDirectlyPatch : BytecodePatch(
         SettingsPatch.PreferenceScreen.MISC.addPreferences(
             SwitchPreference(
                 "revanced_uri_redirect",
-                StringResource("revanced_uri_redirect_title", "Open YouTube links inside app"),
+                StringResource("revanced_uri_redirect_title", "Bypass URL redirect"),
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/openlinksdirectly/patch/OpenLinksDirectlyPatch.kt:40]:
```suggestion
                StringResource("revanced_uri_redirect_title", "Bypass URL redirects"),
```

File: src/main/kotlin/app/revanced/patches/youtube/misc/openlinksdirectly/patch/OpenLinksDirectlyPatch.kt [hunk 3]
```diff
@@ -25,7 +25,7 @@ import org.jf.dexlib2.iface.instruction.formats.Instruction35c
 @Patch
 @DependsOn([IntegrationsPatch::class, SettingsPatch::class])
 @Name("open-links-directly")
-@Description("Bypasses URL redirects and opens links directly inside YouTube app.")
+@Description("Bypasses URL redirects inside YouTube app.")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/openlinksdirectly/patch/OpenLinksDirectlyPatch.kt:28]:
```suggestion
@Description("Bypasses https://youtube.com/redirect URLs.")
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**